app/schemas.py

```diff
@@ -21,6 +22,30 @@ def __init__(self, *args, load_json=False, **kwargs):
         self.load_json = load_json
         super(BaseSchema, self).__init__(*args, **kwargs)
 
+    __envelope__ = {
```

Comment:
This is precursor for wrapping the responses and possible the requests instead of using 'data'
e.g.